packages/server/modules/core/services/admin.ts

```diff
@@ -57,6 +71,31 @@ type AdminProjectListArgs = HasCursor & {
   limit: number
 }
 
+export const adminInviteList = async (
+  args: CollectionQueryArgs
+): Promise<Collection<ServerInviteGraphQLReturnType>> => {
+  const parsedCursor = args.cursor ? parseCursorToDate(args.cursor) : null
```

Comment:
no big deal, but we have some cursor encode/decode utilities that could be used in the implementation - `modules/shared/helpers/graphqlHelper.ts`

Reply:
Good to know